src/tests/FunctionalTests/Android/Device_Emulator/AOT_PROFILED/README.md

```diff
@@ -0,0 +1 @@
+See [Profiled Ahead-Of-Time Compilation on Mono](https://github.com/dotnet/runtime/docs/design/mono/profiled-aot.md#android----running-through-the-android-profiled-aot-functional-test)
```

Comment:
I think that link won't work since it's missing the branch name. Instead you can just use a relative link:
```suggestion
See [Profiled Ahead-Of-Time Compilation on Mono](/docs/design/mono/profiled-aot.md#android----running-through-the-android-profiled-aot-functional-test)
```